village/src/components/SmurfForm.js

```diff
@@ -13,12 +14,18 @@ class SmurfForm extends Component {
   addSmurf = event => {
     event.preventDefault();
```

Comment:
"Add Smurf functional..but weird" I recommend you be more specific in your commits, but I believe you mean to say that add smurf is functional but it only displays the smurf if you reload the page? If so it's because of this event.preventDefault() on line 15. preventDefault() stops the form's default submit event (which is reloading the page) from triggering, in this case though you want that behavior. You need to delete this line in order to have the page reload when you submit a new smurf.